### ISSUE TITLE
Amend Music Industry Specific Items

### ISSUE THREAD

**Issue by dakanji** [zencart/zencart#1102]
Zen Cart comes with some hard coded music industry specific terms which are very confusing for users. I personally still just avoid getting into the whole area myself. I suppose it is meant to somehow represent virtual products of which music downloads were seen as typical in the early days.

Anyway, I propose changing these to more generic terms as follows:

- "Music" -> "Item"
- "Genre" -> "Style"
- "Record Company" -> "Agency"
- "Record Artist" -> "Artist"

"Style" seems to be suitable for "Genre" as it covers this and allows for others
"Agency" is essentially what a record company is and applicable to other types of works
"Artist" would cover "Record Artists" and other types of artists

The one with a question mark is "Item" for "Music" as could be a bit confusing assuming the reading that this is to primarily cover downloadable stuff is correct as there are other "Items" and may be a bit too generic. 

I have done a branch with the changes above implemented but will like some feedback before making a PR if possible.

**Comment by dakanji** [comment]
"Piece" covers "Music" and other works of art. Will change to this while waiting for further feedback.

**Comment by scottcwilson** [comment]
I would personally like to see this removed from the core.  It's incredibly domain specific and irrelevant to a large number of Zen Cart users.

**Comment by zcwilt** [comment]
I realize you have already made a PR, but would like to comment here.
The terms used are generally accepted (having worked in the Music Industry) 

However, having discussed this with Chris, and also based on @scottcwilson comment it may be that we will remove this functionality from core code. 

Moving it to a plugin would also have the advantage of testing hooks in code code, especially catalog side, and may well advise better notifiers/integration for this kind of product type plugin.

**Comment by dakanji** [comment]
No doubt the original terms are the most appropriate for the music industry stuff. 

The confusion (at least in my case when first coming across Zen Cart and till now) is not so much as to how they apply to the music industry setting as to why there is something so related to the music industry sitting in your newly setup cart when all you want to do is sell shoes etc. 

So I thought there might be a wish to broaden the use of that section beyond the music industry as that seemed a bit too narrow a use case to have so much core infrastructure dedicated. 

I had voted with @scottcwilson as well although I read his note as being totally deleting the whole thing altogether. 

A plugin might be a workable solution for those that specifically need this.

I'll close the PR I pushed.
